DuckDuckGo/DBP/DataBrokerProtectionManager.swift

```diff
@@ -72,11 +72,16 @@ public final class DataBrokerProtectionManager {
 }
 
 extension DataBrokerProtectionManager: DataBrokerProtectionDataManagerDelegate {
+
     public func dataBrokerProtectionDataManagerDidUpdateData() {
         loginItemInterface.profileSaved()
     }
 
     public func dataBrokerProtectionDataManagerDidDeleteData() {
         loginItemInterface.dataDeleted()
     }
+
+    public func isAuthenticatedUser() -> Bool {
```

Comment:
Yes, it’s called by the DBP home view controller. Yeah, I know, I had similar thoughts, and thought about combining the methods. However, I don’t want one type (e.g the home view controller) relying on a method that is only there because of a delegate protocol conformance. So, I think having the two is preferable. But…suggestions welcome of course.

Reply:
I don't have any better ideas tbh 😅